src/main/java/org/javarosa/core/model/TriggerableDag.java

```diff
@@ -149,6 +154,7 @@ private QuickTriggerable findTriggerable(Triggerable t) {
      */
     void finalizeTriggerables(FormInstance mainInstance, EvaluationContext ec) throws IllegalStateException {
```

Comment:
I'm guessing this is called when the form is loading/starting up?

Reply:
Yes.